lib/modules/csv_exporter.rb

```diff
@@ -168,11 +164,7 @@ def self.format_row(result)
         self.format_multiple_answer(result.find_demographic_answer_by_key("value_shaping_items"), default),
         result.find_demographic_answer_by_key("shaping_values", default),
         result.find_demographic_answer_by_key("email", default),
-        result.find_demographic_answer_by_key("taken_survey_before", default),
-        result.find_demographic_answer_by_key("wwf_programme", default),
-        result.find_demographic_answer_by_key("wwf_staff_survey", default),
-        result.find_demographic_answer_by_key("ol_pejeta_staff_survey", default),
-        result.language
```

Comment:
If we're removing these, dont forget to remove the partials for the question in the front end!

Reply:
Done!